mmv1/third_party/terraform/tests/resource_monitoring_notification_channel_test.go

```diff
@@ -36,24 +36,41 @@ func TestAccMonitoringNotificationChannel_update(t *testing.T) {
 	})
 }
 
-func TestAccMonitoringNotificationChannel_updateSensitiveLabels(t *testing.T) {
+func TestAccMonitoringNotificationChannel_updateLabels_slack(t *testing.T) {
 	t.Parallel()
+	// being used to skip test if better test skip is located/created feel free to update
+	if os.Getenv(acctest.TestEnvVar) == "" {
+		t.Skipf("Slack auth_token required for test not to fail, skipping test till interal testing slack can be created; use %s=1 to enable", acctest.TestEnvVar)
+	}
```

Comment:
If we want the test to not run in the nightly tests I think using the `acctest.SkipIfVcr` function ([link](https://github.com/hashicorp/magic-modules/blob/4ace2e99407ec6f46cceec726d1f8ffd132c6147/mmv1/third_party/terraform/acctest/provider_test_utils.go#L270)) would be the best way to selectively skip the Slack acceptance tests until a working auth token is available for the test to use.

Reply:
Thanks Sarah ill update this